diff --git implot.cpp implot.cpp
index 91f3405..004d902 100644
--- implot.cpp
+++ implot.cpp
@@ -124,7 +124,9 @@ You can read releases logs https://github.com/epezent/implot/releases for more d
 
 */
 
+#ifndef IMGUI_DEFINE_MATH_OPERATORS
 #define IMGUI_DEFINE_MATH_OPERATORS
+#endif
 #include "implot.h"
 #include "implot_internal.h"
 
diff --git implot_items.cpp implot_items.cpp
index c57ca9d..06ad121 100644
--- implot_items.cpp
+++ implot_items.cpp
@@ -22,7 +22,9 @@
 
 // ImPlot v0.16
 
+#ifndef IMGUI_DEFINE_MATH_OPERATORS
 #define IMGUI_DEFINE_MATH_OPERATORS
+#endif
 #include "implot.h"
 #include "implot_internal.h"
 